src/components/csvLocalizer/jsonLocalizer.js

```diff
@@ -23,12 +23,12 @@ const JsonLocalizer = () => {
   const [data, setData] = React.useState({});
   const [chosenFiles, setChosenFiles] = React.useState([en, de, fr, nl]);
   const [activeCellId, setActiveCellId] = React.useState('');
-  const [newTranslation, setNewtranslation] = React.useState('');
+  const [alert, setAlert] = React.useState('');
 
-  const inputRef = React.useRef();
+  const textareaRef = React.useRef();
 
   React.useEffect(() => {
-    inputRef.current?.focus();
+    textareaRef.current?.focus();
```

Comment:
Will it try to focus the textarea when the `activeCellId` is set to empty string?

Reply:
Yes, it's works fine with empty string.